yarn.lock

```diff
@@ -71,7 +71,19 @@
   dependencies:
     yaml "1.10.2"
 
-"@dcl/schemas@3.1.1":
+"@dcl/content-validator@^1.0.0-20220106210721.commit-0a7b4e5":
+  version "1.0.0-20220106210721.commit-0a7b4e5"
+  resolved "https://registry.yarnpkg.com/@dcl/content-validator/-/content-validator-1.0.0-20220106210721.commit-0a7b4e5.tgz#c2b02438771ae89a3e153321bde3a11acec27799"
+  integrity sha512-4iFaWS5EgEEJCvx4JAnGnxxsKSTuQuj4yT6fIwENvtFsCRS2cQLGtmKn0veKTk5zXCJjK9JCPphisl5iwMkmgA==
+  dependencies:
+    "@dcl/schemas" "^3.1.1"
+    "@dcl/urn-resolver" "^1.3.0"
+    dcl-catalyst-commons "7.1.4-20220105153139.commit-79398b8"
+    ethers "^5.5.2"
```

Comment:
Out of this PR, is it necessary that `@dcl/content-validator` uses ethers?

Reply:
it is used for one validation to check an address is a valid Ethereum address, may be I can replace it with a regex

Reply:
https://github.com/decentraland/eth-connect/blob/b0601293eda8d644f57259b48f5a991953eb3fa0/src/utils/utils.ts#L515-L551

Reply:
Already replace `ethers` with `@ethersproject/address` in this [commit](https://github.com/decentraland/content-validator/commit/e97b937816f355e1f1bfefc746a8963ed7aaf35e). What do you think?